### DISCUSSION TITLE
Are all islands valley bottom?

### DISCUSSION THREAD

**Discussion by philipbaileynar** [Riverscapes/riverscapes-tools#144] [VBET] [answered]
We currently rasterize the flow lines such that cells inside the bankfull polygon are 1 and those outside are 0. @KellyMWhitehead is enhancing this to run a Euclidean distance spreading out from the bankfull polygon so that we can taper the likelihood with distance.

But it occurs to me that any "donuts" in the rastersized flow lines (I use quotes because it's a raster and so they are not technically polygon holes) are almost certainly valley bottom. They represent bars, islands etc. We can maybe assign the cells in these donuts as 1 or 0.9 etc?

Here's an example in the Klamath. Is every area enclosed by the NHD flow lines (red) likely to be valley bottom?

<img width="1138" alt="Screen Shot 2021-01-14 at 9 15 27 AM" src="https://user-images.githubusercontent.com/8645379/104624972-116eaf80-5649-11eb-82de-5e7ef2e626db.png">

Could a contained raster donut ever not be valley bottom? Is there some size above which they should not be considered valley bottom? @joewheaton I'm thinking of those volcanic calderas just north of Yosemite?

**Comment by joewheaton** [comment, marked as answer]
> Could a contained raster donut ever not be valley bottom?

For the resolution of these tools that is generally true. They are typically floodplain islands and rarely bars. We can have donuts occasionally in some odd geologic areas around outcrops that are NOT valley bottom,  and occasionally around island terraces. However, these are relatively minor problems. 

As far as:

>  Is there some size above which they should not be considered valley bottom? @joewheaton I'm thinking of those volcanic calderas just north of Yosemite?

That's a good question. However, I think this is dangerous. It has less to do with the size, and more to do with the HAND results in that donut. I would be a fan of just getting rid of the donuts unless the HAND values are noticeably higher than floodplain HAND values for rest of candidate valley bottom.  Can of worms?

**Comment by joewheaton** [comment]
@shelbysawyer you should be tracking this one too and asking questions... or commenting.
